Resources/Prototypes/Entities/Clothing/Hands/gloves.yml

```diff
@@ -227,7 +227,7 @@
   parent: ClothingHandsBase
   id: ThievingGloves
   name: black gloves #Intentionally named after regular gloves, they're meant to be sneaky.
-  description: Seemingly regular black gloves. The fingertips are outfitted with nanotech that makes stealing a breeze.
+  description: Regular black gloves that do not keep you from frying. #Same description as normal gloves for the above reason
```

Comment:
This should have some kind of detailed examine because there's no way to distinguish these in normal gameplay.

Reply:
That's in line with other disguised uplink items. You can distinguish them by trying to use them, same as noslips, etc.

Reply:
I'm totally open to dropping this change but yeah this is the same way that other disguised uplink items are handled. Also how I assume this one was supposed to be because otherwise there's not much point in changing the name

Reply:
> That's in line with other disguised uplink items. You can distinguish them by trying to use them. Same as noslips, etc.

noslips doesn't require you to try taking items off of someone else to know if it's going to be stealthy or not.

Reply:
Alright, I just went ahead and reverted it back to what it was.